test/unit/VerifierRegistry.js

```diff
@@ -52,12 +53,20 @@ contract('VerifierRegistry', (accounts) => {
     });
 
     it('should require verifier to not be already created', async () => {
-      await verifierRegistryContract.create('127.0.0.1')
+      await verifierRegistryContract.create('mike', '127.0.0.1')
         .catch((error) => {
           assert.isDefined(error);
         });
     });
 
+    it('should provide some unique name', async () => {
```

Comment:
This test will be always true, because there is`beforeEach` that creating new contract for each test. You probably should change `beforeEach()` => `before()`